pcm_osx.cpp

```diff
@@ -17,6 +17,7 @@ class PcmOutputMacOSX: public PcmOutput {
     };
     short buffer[BUFFER_SIZE];
     int index;
+    const uint32_t sample_rate;
```

Comment:
It's good practice to put constant instance data such as this `sample_rate` member at the top of the `private` section. This helps to logically organize the instance data, rather than mixing up constant and mutable data together.